src/openrct2/ride/TrackData.cpp

```diff
@@ -5427,6 +5427,765 @@ constexpr static uint8_t TrackTypeToSpinFunction[TrackElemType::Count] = {
+        case TrackElemType::RightQuarterTurn3TilesCovered:
+        case TrackElemType::RightCurvedLiftHill:
+            return EvaluatorConst<-59>;
+        case TrackElemType::LeftBankedQuarterTurn3Tiles:
+        case TrackElemType::LeftHalfBankedHelixUpSmall:
+        case TrackElemType::LeftHalfBankedHelixDownSmall:
+            return EvaluatorConst<100>;
+        case TrackElemType::RightBankedQuarterTurn3Tiles:
+        case TrackElemType::RightHalfBankedHelixUpSmall:
+        case TrackElemType::RightHalfBankedHelixDownSmall:
+            return EvaluatorConst<-100>;
+        case TrackElemType::LeftQuarterTurn1Tile:
+            return EvaluatorConst<45>;
+        case TrackElemType::RightQuarterTurn1Tile:
+            return EvaluatorConst<-45>;
+        case TrackElemType::HalfLoopUp:
+        case TrackElemType::FlyerHalfLoopUp:
+            return EvaluatorConst<0>;
+        case TrackElemType::HalfLoopDown:
+        case TrackElemType::FlyerHalfLoopDown:
+            return EvaluatorConst<0>;
+        case TrackElemType::LeftCorkscrewUp:
+        case TrackElemType::RightCorkscrewDown:
+        case TrackElemType::LeftFlyerCorkscrewUp:
+        case TrackElemType::RightFlyerCorkscrewDown:
+            return EvaluatorConst<70>;
+        case TrackElemType::RightCorkscrewUp:
+        case TrackElemType::LeftCorkscrewDown:
+        case TrackElemType::RightFlyerCorkscrewUp:
+        case TrackElemType::LeftFlyerCorkscrewDown:
+            return EvaluatorConst<-70>;
+        case TrackElemType::FlatToUp60:
+        case TrackElemType::Down60ToFlat:
+            return EvaluatorConst<0>;
+        case TrackElemType::Up60ToFlat:
+        case TrackElemType::FlatToDown60:
+        case TrackElemType::BrakeForDrop:
+            return EvaluatorConst<0>;
+        case TrackElemType::LeftQuarterTurn1TileUp60:
+        case TrackElemType::LeftQuarterTurn1TileDown60:
+            return EvaluatorConst<88>;
+        case TrackElemType::RightQuarterTurn1TileUp60:
+        case TrackElemType::RightQuarterTurn1TileDown60:
+            return EvaluatorConst<-88>;
+        case TrackElemType::Watersplash:
+            return EvaluatorConst<0>;
+        case TrackElemType::FlatToUp60LongBase:
+        case TrackElemType::Down60ToFlatLongBase:
+            return EvaluatorConst<0>;
+        case TrackElemType::Up60ToFlatLongBase:
+        case TrackElemType::FlatToDown60LongBase:
+            return EvaluatorConst<0>;
+        case TrackElemType::ReverseFreefallSlope:
+        case TrackElemType::AirThrustVerticalDownToLevel:
+            return EvaluatorConst<0>;
+        case TrackElemType::Up60ToUp90:
+        case TrackElemType::Down90ToDown60:
+            return EvaluatorConst<0>;
+        case TrackElemType::Up90ToUp60:
+        case TrackElemType::Down60ToDown90:
+            return EvaluatorConst<0>;
+        case TrackElemType::LeftEighthToDiag:
+        case TrackElemType::LeftEighthToOrthogonal:
+            return EvaluatorConst<0>;
```

Comment:
This is not right. Previously `lateralFactor` was assigned 137.
```suggestion
            return EvaluatorConst<137>;
```